### PR TITLE
unit-tests added

### PR DESCRIPTION
**Pull request by SpiderDan98**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
   ],
   "require": {
     "php": ">=7.1",
-    "spryker/company-business-unit": "^2.0.0"
+    "spryker/company-business-unit": "^1.0.0 || ^2.0.0",
```
Comment (SpiderDan98) [inline composer.json:17]: Checked for possible versions

File: src/FondOfSpryker/Client/CompanyBusinessUnitsRestApi/CompanyBusinessUnitsRestApiFactory.php [hunk 2]
```diff
@@ -18,6 +18,8 @@ public function createZedCompanyBusinessUnitsRestApiStub(): CompanyBusinessUnits
     }
 
     /**
+     * @throws
```
Comment (daniel-rose) [inline src/FondOfSpryker/Client/CompanyBusinessUnitsRestApi/CompanyBusinessUnitsRestApiFactory.php]: remove @throws

File: src/FondOfSpryker/Zed/CompanyBusinessUnitsRestApi/Business/CompanyBusinessUnitsRestApiBusinessFactory.php [hunk 3]
```diff
@@ -40,6 +40,8 @@ public function createCompanyBusinessUnitWriter(): CompanyBusinessUnitWriterInte
     }
 
     /**
+     * @throws
```
Comment (daniel-rose) [inline src/FondOfSpryker/Zed/CompanyBusinessUnitsRestApi/Business/CompanyBusinessUnitsRestApiBusinessFactory.php]: remove @throws

File: src/FondOfSpryker/Zed/CompanyBusinessUnitsRestApi/Business/CompanyBusinessUnitsRestApiBusinessFactory.php [hunk 4]
```diff
@@ -48,6 +50,8 @@ protected function getCompanyBusinessUnitMapperPlugins(): array
...
```
Comment (daniel-rose) [inline src/FondOfSpryker/Zed/CompanyBusinessUnitsRestApi/Business/CompanyBusinessUnitsRestApiBusinessFactory.php]: remove @throws

**Review by daniel-rose — CHANGES_REQUESTED** [review]

**Review by daniel-rose — APPROVED** [review]

**Pull request merged**